backend/app/tests/api/routes/test_onboarding.py

```
@@ -158,3 +172,203 @@ def test_onboard_project_with_auto_generated_defaults(
+        json=onboard_data,
+        headers=superuser_token_headers,
+    )
+
+    assert response.status_code == 409
+    error_response = response.json()
+    assert "error" in error_response
+    assert "Project already exists" in error_response["error"]
+
+
+def test_onboard_project_invalid_provider(
+    client: TestClient, superuser_token_headers: dict[str, str], db: Session
```

Ruff (0.14.5)

Unused function argument: `db`

(ARG001)